flink-runtime/src/main/java/org/apache/flink/runtime/asyncprocessing/AsyncExecutionController.java

```diff
@@ -51,15 +56,24 @@ public class AsyncExecutionController<K> {
 
     private static final Logger LOG = LoggerFactory.getLogger(AsyncExecutionController.class);
 
-    public static final int DEFAULT_BATCH_SIZE = 1000;
-    public static final int DEFAULT_MAX_IN_FLIGHT_RECORD_NUM = 6000;
+    private static final int DEFAULT_BATCH_SIZE = 1000;
+
+    private static final int DEFAULT_BUFFER_TIMEOUT = 1000;
```

Comment:
How about remove the constants and use `ASYNC_STATE_BUFFER_SIZE.defaultValue()` from the `ExecutionConfig`? This can help avoid maintaining the same value in two places.